src/loaders/express.js

```diff
@@ -14,34 +15,42 @@ module.exports = async (app) => {
   // Enable Cross Origin Resource Sharing to all origins by default
   app.use(cors());
 
-  // Middleware that transforms the raw string of req.body into json
-
   // The custom limits are required so that SNS topics can submit work results
   // up to the size of the max SNS topic limit (256k), it defaults to 100kb.
   app.use(bodyParser.urlencoded({ extended: false, limit: '1mb', parameterLimit: 300000 }));
   app.use(bodyParser.text({ extended: false, limit: '1mb', parameterLimit: 300000 }));
   app.use(bodyParser.json({ extended: false, limit: '10mb', parameterLimit: 300000 }));
 
+  // Enable AWS XRay
+  // eslint-disable-next-line global-require
+  AWSXRay.captureHTTPsGlobal(require('http'));
+  app.use(AWSXRay.express.openSegment(`API-${config.clusterEnv}-express`));
+
   app.use(OpenApiValidator.middleware({
     apiSpec: path.join(__dirname, '..', 'specs', 'api.yaml'),
     validateRequests: true,
     validateResponses: true,
     operationHandlers: path.join(__dirname, '..', 'api'),
   }));
 
+  app.use(AWSXRay.express.closeSegment());
+
```

Comment:
Hooks to enable XRay on the Express side as well. This is a supported pattern, so no need to refactor anything.